grid.css

```diff
@@ -10,57 +10,79 @@ heading1, heading2{
 /* style for the headings section */
 heading1{
 	padding: 0;
-    border-bottom: 1px solid white;
+    border-bottom: 0.2vw solid #ffffff;
 	grid-template-rows: 100%;
 	background-color: #292a33;	
   	text-align: center;
 	position: sticky;
 	top: 0vw;
+    box-shadow: 0 10px 6px -6px #18191f;
```

Comment:
#added box shadow to contrast and stand out more when users scroll through song outputs